LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
According to https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Licensing this file is not necessary, and it is sufficient to declare the licence in the DESCRIPTION file.

I would prefer not to have the LICENSE file, and keep things as they are now, i.e., to have only the licence reference in the DESCRIPTION file. It seems easier and simpler. 

In case you prefer to add the LICENSE file, the description file has to be changed:
...
LICENSE: file LICENSE 
...